README.md

```diff
@@ -125,3 +129,22 @@ jobs:
           configurationFile: example/renovate-config.js
           token: 'x-access-token:${{ steps.get_token.outputs.app_token }}'
 ```
+
+## Troubleshooting
+
+### Debug Logging
+In case of issues, it's always a good idea to enable debug logging first.
+To enable debug logging, add the environment variable `LOG_LEVEL: 'debug'` to the action:
+```yml
+      - name: Self-hosted Renovate
+        uses: renovatebot/github-action@v21.30.0
+        with:
+          configurationFile: example/renovate-config.js
+          token: ${{ secrets.RENOVATE_TOKEN }}
+        env:
+          LOG_LEVEL: 'debug'
+```
+
+### Special token requirements when using the `github-actions` manager
+
+If you want to use the `github-actions` [manager](https://docs.renovatebot.com/modules/manager/github-actions/) in renovate, ensure that the `token` you provide contains the `workflow` scope. Otherwise, Github does not allow Renovate to update worklow files and therefore it will be unable to create update PR's for affected packages (like `actions/checkout` or `renovatebot/github-action` itself).
```

Comment:
```suggestion
If you want to use the `github-actions` [manager](https://docs.renovatebot.com/modules/manager/github-actions/) in Renovate, ensure that the `token` you provide contains the `workflow` scope.
Otherwise, GitHub does not allow Renovate to update worklow files and therefore it will be unable to create update PRs for affected packages (like `actions/checkout` or `renovatebot/github-action` itself).
```